internal/handlers/publishers.go

```diff
@@ -19,62 +17,56 @@ func GetPublishers(ctx *fiber.Ctx) error {
 }
 
 func GetPublisher(ctx *fiber.Ctx) error {
-	var publisher models.Publisher
-
-	if err := db.Database.First(&publisher, ctx.Params("id")).Error; err != nil {
-		if errors.Is(err, gorm.ErrRecordNotFound) {
-			return ctx.Status(fiber.StatusNotFound).JSON(fiber.Map{
-				"message": err.Error(),
-			})
-		}
+	publisherEntity := models.Publisher{}
```

Comment:
```suggestion
	publisher := models.Publisher{}
```

This is the only place where we call it `publisherEntity`.